### PR TITLE
dockerize frontend

### LINKED ISSUE #23: * set up starter project
**we should set up the project before start working on any issue**
- ask TA about the project in more details
- find the best starter kit from open source code base
- It should have the highest priority right now

WIP: init server

### PR DESCRIPTION
**Pull request by PENGYUXIONG**
- add in docker-compose.yml, Dockerfile, partially adapted makefile from Ankush's previous commits
- node version is node:alpine
- have empty folders: components, actions, actionTypes to hold the react elements

### CODE REVIEW

**Comment by PENGYUXIONG** [comment]
Please review and give comments. Approve if you think it is all good, I will merge this pr after get everyone's approval.

thx :)

**Review by AnkushSharma2698 — APPROVED** [review]
looks good!

**Review by KevinHQi — APPROVED** [review]

**Pull request merged**
